src/web/mjs/connectors/kuaikanmanhua.mjs

```diff
@@ -78,13 +78,21 @@ export default class Kauikanmanhua extends Connector {
     }
 
     async _getPageList(manga, chapter, callback) {
+        // clone header, then set mobile user-agent.
+        // mobile site gives larger (1280px wide) images, while desktop site returns smaller (750px) images.
```

Comment:
Redundant comment, commit messages should be enough to trace the reason for change